isis/src/mgs/apps/mocuncompress/main.cpp

```diff
@@ -128,6 +148,303 @@ enum MocCompressEnum {
   RAW = 0, PRED, XFORM
 } MocCompress = RAW;
 
+
+
+extern void exit();
+
+#define BUFFERSIZE  4096
+
+/* Huffman tree (in table form) */
+uint8 code[256];
+uint8 left[256];
+uint8 right[256];
+
+int pred_past_eof;
+
+
+
+void decodeLoad(char *decodefile)
```

Comment:
Stuffed some floating functions in the app folder into main as they were only being used here.